src/packages/ellipsis/ellipsis.taro.tsx

```
@@ -2,7 +2,7 @@
 import { nextTick, createSelectorQuery } from '@tarojs/taro'
 import classNames from 'classnames'
 import { View } from '@tarojs/components'
-import { getRectInMultiPlatform } from '@/utils/taro/get-rect'
+import { getRectInMultiPlatform, getRectInMultiPlatformWithoutCache } from '@/utils/taro/get-rect'
```

ESLint

[error] Replace `·getRectInMultiPlatform,·getRectInMultiPlatformWithoutCache·` with `⏎··getRectInMultiPlatform,⏎··getRectInMultiPlatformWithoutCache,⏎`

(prettier/prettier)